src/components/Loading/Loading.module.css

```diff
@@ -0,0 +1,37 @@
+
+div .loading-container {
```

Comment:
Why are we using the div selector? I believe using the className should suffice.